### ISSUE TITLE
Automatic switching between menus at certain time intervals

### ISSUE THREAD

**Issue by ahmtsenlik** [GeekMagicClock/smalltv-pro#91]
It would be much better if it would switch automatically instead of changing between menus by touching or via the interface. For example, 10 minutes weather, 10 minutes bitcoin, 15 minutes stock, etc.

**Comment by GeekMagicClock** [comment]
@ahmtsenlik  Good idea, worth a try, and also I was thinking of a display schdule, In the 9:00 AM display "weather" for a certern time, and then 10:00 AM display "bitcoin" for anther period time, 7:00 PM for another theme, what do you think?

**Comment by ahmtsenlik** [comment]
@GeekMagicClock Your idea is also a different perspective, you can do both but I still think the idea I mentioned should be a priority, not everyone wants to define a specific time frame.

**Comment by nate922** [comment]
@GeekMagicClock I agree with @ahmtsenlik on this, would be great to cycle clock/weather for 30s, bitcoin price for 30s etc

**Comment by arsophan** [comment]
Cycling through certain apps would be a beautiful feature. My personal configuration would be: random picture (maybe another feature needed "start pictures with a random pic"), after that show the weather app for a while until next random pic. Otherwise the pic app is not much used since it needs user action.
